security/pkg/server/ca/server.go

```diff
@@ -140,11 +141,20 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 	if len(rootCertBytes) != 0 {
 		respCertChain = append(respCertChain, string(rootCertBytes))
 	}
-	response := &pb.IstioCertificateResponse{
-		CertChain: respCertChain,
+
+	// flatten `respCertChain` since each element might be a concatenated multi-cert PEM
```

Comment:
very nitpicky, but this seems the opposite of flattening, we are expanding it?

Reply:
haha I'm thinking of "flattening" in the sense similar to a nested-array, but yes expanding is more accurate